src/model/guild/mod.rs

```diff
@@ -131,6 +131,9 @@ pub struct Guild {
     /// The Id of the [`User`] who owns the guild.
     pub owner_id: UserId,
     /// A mapping of [`User`]s' Ids to their current presences.
+    ///
+    /// This will be empty unless the "guild presences" privileged intent is
```

Comment:
This is quite the good clue! That's why I would add the **Info** label.
```suggestion
    /// **Info**:
    /// This will be empty unless the "guild presences" privileged intent is
```

Reply:
🕵️ Should it be info or note? I see note used quite more often.

Reply:
🕵️ That is a good idea, we can use `**Note**` instead!